package-lock.json

```diff
@@ -144,17 +403,23 @@
       "integrity": "sha512-5teOsQWABXHHBFP9y3skS5P3d/WfWXpv3FUpy+LorMrNYaT9pI4oLMQX7jzQ2KklNpGpWHzdCXTDT2Y3XGlZBw==",
       "dev": true,
       "requires": {
-        "micromatch": "3.1.10",
-        "normalize-path": "2.1.1"
+        "micromatch": "^3.1.4",
```

Comment:
Are some of these required versions going backward because they did some improvements to the library that make it compatible with more versions of the dependencies?

Reply:
Ya, I wouldn't ever do code review on the `package-lock.json` file. That file is just auto-generated from NPM. Don't look at it.